### ISSUE TITLE
Cluster info & leaf node details 

### ISSUE THREAD

**Issue by darjisanket** [nats-nui/nui#31]
I am exploring NATS, and nats-nui has been helpful in managing things through the UI. I believe that adding cluster information and leaf node details would improve the ability to manage the cluster runtime effectively

**Comment by pricelessrabbit** [comment]
yep, for sure add some metrics capabilities in the UI is in the roadmap, both to see cluster info and client connections, but we have limited time-bandwidth 🐰
